### PR TITLE
[13.0][MIG] printer_zpl2

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Forward port of https://github.com/OCA/report-print-send/pull/182

### CODE REVIEW

**Comment by fmdl** [comment]
@schout-it I have migrate this module in V12, I have plan to add button on label component to change quickly the position of component please find my PR for V12 here : https://github.com/OCA/report-print-send/pull/205

**Comment by AaronHForgeFlow** [comment]
@fmdl I included your commit here. I will attend failing tests as soon as possible.

File: printer_zpl2/views/printing_label_zpl2.xml [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+                        <button
+                            name="unlink_action"
+                            string="Remove from the 'Action' menu"
+                            type="object"
+                            attrs="{'invisible':[('action_window_id','=',False)]}"
+                            icon="fa-minus-square"
+                            help="Remove the contextual action."
+                            class="oe_stat_button"
+                        />
+                        <button
```
Comment (fmdl) [inline printer_zpl2/views/printing_label_zpl2.xml]: replace by in `sheet`
```xml
<field name="active" invisible="1"/>
<widget name="web_ribbon" text="Archived" bg_color="bg-danger" attrs="{'invisible': [('active', '=', True)]}"/>
```

Comment (AaronHForgeFlow) [reply printer_zpl2/views/printing_label_zpl2.xml]: Done. Thank you!

**Comment by fmdl** [comment]
@AaronHForgeFlow my commit (quick move doesn't work correctly with labelary), please find the correct commit : https://github.com/OCA/report-print-send/pull/205/commits/3e9bb519508436ba667c7e54705d42db6375ed19

File: printer_zpl2/views/printing_label_zpl2.xml [hunk 2]
```diff
@@ -22,6 +22,13 @@
                 </header>
                 <sheet>
                     <div class="oe_button_box" name="button_box">
+                        <widget
```
Comment (fmdl) [inline printer_zpl2/views/printing_label_zpl2.xml]: do put inside the `div class oe_button_box`, juste inside `sheet`

**Comment by AaronHForgeFlow** [comment]
Ops, it seems I made a mistake when rebasing. I will fix this.

File: printer_zpl2/views/printing_label_zpl2.xml (hunk 1 above)
Comment (AaronHForgeFlow) [reply printer_zpl2/views/printing_label_zpl2.xml]: > @AaronHForgeFlow my commit (quick move doesn't work correctly with labelary), please find the correct commit : [3e9bb51](https://github.com/OCA/report-print-send/commit/3e9bb519508436ba667c7e54705d42db6375ed19)

Thanks for the update. I will do so when your PR is merged. I have many conflicts when cherry-picking your commit in 13.0 branch, so I prefer to fix those conflicts just once.

**Comment by AaronHForgeFlow** [comment]
Github is not allowing me to reopen this PR. I will continue here: https://github.com/OCA/report-print-send/pull/210

**Pull request closed without merging**